framework/src/modules/http_api/controllers/transactions.js

```diff
@@ -138,19 +138,24 @@ TransactionsController.postTransaction = function(context, next) {
 	return channel.invoke('chain:postTransaction', [
 		transaction,
 		(err, data) => {
-			if (err) {
-				if (err instanceof ApiError) {
-					context.statusCode = err.code;
-					delete err.code;
-				}
+			let error = null;
+
+			if (data.success) {
+				return next(null, {
+					data: { message: 'Transaction(s) accepted' },
+					meta: { status: true },
+				});
+			}
 
-				return next(err);
+			if (err) {
+				error = new ApiError(err, apiCodes.PROCESSING_ERROR);
+			} else {
+				error = new ApiError(data.message, apiCodes.PROCESSING_ERROR);
```

Comment:
should this be both `PROCESSING_ERROR`?

Reply:
This code comes from https://github.com/LiskHQ/lisk/blob/dfc015b200c816826826cac03dc6575726849ca2/framework/src/modules/chain/modules/transactions.js#L631 so the logic has been just moved to HTTP API module. I am unaware if both should be `PROCESSING_ERROR` and in that case we could address this in a separate issue.